addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -121,6 +121,10 @@ def _create_invoice(self, order, so_line, amount):
                     subtype_id=self.env.ref('mail.mt_note').id)
         return invoice
 
+    @api.model
```

Comment:
@qdp-odoo I test without the "action_move_create" method
We can't reconciliate with the sale order because we don't have any move line created return by the method (I add a raise to remove after your check).
Can you check.
Thanks